Upstream: Yes
Reason: undefined references to dl* functions
Source: Daniel Mierswa <impulze@impulze.org>

http://lists.shmoo.com/pipermail/hostap/2009-February/019212.html

diff --git a/wpa_supplicant/Makefile b/wpa_supplicant/Makefile
index 65adf53..5002c56 100644
--- a/wpa_supplicant/Makefile
+++ b/wpa_supplicant/Makefile
@@ -566,6 +566,7 @@ OBJS += ../src/eap_peer/tncc.o
 OBJS_h += ../src/eap_server/eap_tnc.o
 OBJS_h += ../src/eap_server/tncs.o
 NEED_BASE64=y
+LIBS += -ldl
 endif
 
 ifdef CONFIG_IEEE8021X_EAPOL
